intelmq/bots/experts/taxonomy/expert.py

```diff
@@ -44,24 +45,25 @@
     "unauthorized-command": "intrusions",  # not in ENISA eCSIRT-II taxonomy
     "unauthorized-login": "intrusions",  # not in ENISA eCSIRT-II taxonomy
     "unprivileged-account-compromise": "intrusions",
-    "c2server": "malicious code",  # ENISA eCSIRT-II taxonomy: 'c2server'
-    "dga domain": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "infected-system": "malicious code",  # ENISA eCSIRT-II taxonomy: 'infected-system'
-    "malware": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "malware-configuration": "malicious code",  # ENISA eCSIRT-II taxonomy: 'malware-configuration'
-    "malware-distribution": "malicious code",
-    "ransomware": "malicious code",  # not in ENISA eCSIRT-II taxonomy
-    "blacklist": "other",
-    "other": "other",  # not in ENISA eCSIRT-II taxonomy
+    "c2-server": "malicious-code",
+    "dga-domain": "malicious-code",  # not in ENISA eCSIRT-II taxonomy
```

Comment:
as long as this type is not in RSIT we don't know the final name anyway. to avoid double renaming, I change it back here for now